src/nupic/bindings/sparse_matrix.i

```diff
@@ -27,16 +27,24 @@
 #include <nupic/math/NearestNeighbor.hpp>
 #include <nupic/proto/SparseMatrixProto.capnp.h>
 #include <nupic/proto/SparseBinaryMatrixProto.capnp.h>
-#if !CAPNP_LITE
 #include <nupic/py_support/PyCapnp.hpp>
-#endif
 #include <nupic/py_support/NumpyVector.hpp>
 #include <nupic/py_support/PythonStream.hpp>
 
 %}
 
 %pythoncode %{
 import numbers
+
+try:
+	# NOTE need to import capnp first to activate the magic necessary for
```

Comment:
You ended up with tab characters here and elsewhere in this file. Please replace with 2-space indents